idc/views.py

```diff
@@ -264,11 +284,29 @@ def explore_data_page(request):
         collapse_on = req.get('collapse_on', 'SeriesInstanceUID')
         is_json = (req.get('is_json', "False").lower() == "true")
         uniques = json.loads(req.get('uniques', '[]'))
+        totals = json.loads(req.get('totals', '[]'))
+
         record_limit = int(req.get('record_limit', '2000'))
         offset = int(req.get('offset', '0'))
-
+        #sort_on = req.get('sort_on', collapse_on+' asc')
+        cohort_id = req.get('cohort_id','-1')
+
+        cohort_filters={}
+        if (int(cohort_id)>-1):
+            cohort = Cohort.objects.get(id=cohort_id, active=True)
+            cohort.perm = cohort.get_perm(request)
```

Comment:
You can ask for the case, study, and series count here; the names of the attributes are:
- case_count
- series_count
- study_count
- collections: This is a string list of the collections with data in this cohort